scripts/test-py3.sh

```diff
@@ -5,14 +5,12 @@ pytest openlibrary/catalog openlibrary/coverstore openlibrary/mocks openlibrary/
        --ignore=openlibrary/catalog/marc/tests/test_get_subjects.py \
        --ignore=openlibrary/catalog/marc/tests/test_parse.py \
        --ignore=openlibrary/tests/catalog/test_get_ia.py \
-       --ignore=openlibrary/coverstore/tests/test_doctests.py \
```

Comment:
I think you're removing this line because removing the _ignore_ enables these tests? If that's the right understanding, excellent. (New to this codebase, so I want to confirm my understanding sometimes.)

Reply:
👍 Exactly! The first line, adds the whole directory (`pytest openlibrary/catalog openlibrary/coverstore ...`), and then this line excludes this file. Since this files is fixed, it no longer needs to be excluded! 🎉 

Note that this file, `test-py3.sh`, is a temporary file while we do the py2-3 migration. The full test suite is done by `make test`. This file runs the tests which we know work on python 3 to make sure we don't regress :P It also runs a few tests (below in the file) which won't fail the build, but display fail-state inside Travis.